src/modules/functionality.js

```diff
@@ -0,0 +1,33 @@
+import Work from './work.js';
+import LocalStorage from './localstorage.js';
+
+let list;
+if (LocalStorage.getData() === null) {
+  list = [];
+} else {
+  list = LocalStorage.getData();
+}
+export default class functions {
+    static addtask = (inputTask) => {
+      if (inputTask) {
+        const newtask = new Work(inputTask);
+        list.push(newtask);
+        list.forEach((item, number) => {
+          item.index = number;
+        });
+        LocalStorage.saveData(list);
+      }
+    }
```

Comment:
[ Optional Changes ]

I am marking this change as optional because the previous reviewer did not mention it but it's supposed to be required. Therefore, I highly recommend you work on it before your next milestone.

- [ ] When adding a todo, it's said that if a 5th task is added to the list, the index of that task should equal 5.

<img width="1042" alt="Screenshot 2023-01-20 at 8 56 52 AM" src="https://user-images.githubusercontent.com/50941074/213656451-a4401ee2-11c0-4e5d-bcfe-56343a4af498.png">

- [ ] In your case right now, I added 6 todos, and the 6th todo's index shows 5. It's supposed to show 6. You can achieve that by adding plus 1 to the todo index in the single todo object that you push to the array when adding. 👍🏽 

<img width="468" alt="Screenshot 2023-01-20 at 9 04 09 AM" src="https://user-images.githubusercontent.com/50941074/213657316-6281083a-77c0-4486-aa37-08f7e03b118f.png">

Reply:
hey there, please I wasn't able to implement this because the indices weren't delete unless they are indexed from Zero